package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-http-auth",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
>If I understand correctly, the NPM way is to keep the old version number up until next release, is that correct?

Correct. If people want the latest and greatest, they would just set their package manager to get "#master" (NPM), "@dev" (Composer) or whatever is the practise in their manager. They can also refer to a certain commit hash to get exactly the "snapshot" they want.

So in github version numbers are for pre-releases or releases only.

https://docs.npmjs.com/cli/install